tests/language-feature/zero-initialize/struct-no-zero-init.slang

```diff
@@ -0,0 +1,37 @@
+//TEST:SIMPLE(filecheck=CHECK): -target hlsl -stage compute -entry computeMain
+//TEST:SIMPLE(filecheck=CHECK): -target glsl -stage compute -entry computeMain
+//TEST:SIMPLE(filecheck=CHECK): -target glsl -stage compute -entry computeMain -DFAIL
+
+// CHECK-NOT: {{.* }}= 0;
+
+RWStructuredBuffer<int> outputBuffer;
+
+struct MyStruct_base
+{
+    int a;
+};
+struct MyStruct : MyStruct_base
+{
+    int b;
+    int c;
+};
+MyStruct getStruct()
+{
+    MyStruct myStruct;
+    return myStruct;
+}
+[numthreads(1, 1, 1)]
+void computeMain(int3 dispatchThreadID: SV_DispatchThreadID)
+{
+    MyStruct myStruct = getStruct();
+#ifdef fail
+    myStruct.
+#endif
```

Comment:
Not sure what these is for.
It might be related to the line 3 where "FAIL" is defined?

Reply:
I was testing the language server and was going to add a language server test, but realized it just works as is.